src/main/java/com/ecommerce/controller/OrderController.java

```diff
@@ -45,10 +45,17 @@ public String showAllOrders(
         return "user/order/index";
     }
 
+    @GetMapping({"/payment", "/new"})
+    public String showPaymentForm(@ModelAttribute("order") @Valid OrderDTO orderDTO, Model model) {
+        orderDTO = orderService.initOrder(orderDTO);
+        model.addAttribute("order", orderDTO);
+        return "user/order/payment";
+    }
+
     @PostMapping
-    public String createNewOrder(@ModelAttribute @Valid OrderDTO orderDTO, BindingResult result, final RedirectAttributes redirectAttributes) {
+    public String createNewOrder(@ModelAttribute("order") @Valid OrderDTO orderDTO, BindingResult result, final RedirectAttributes redirectAttributes) {
         if (result.hasErrors()) {
-            return "user/order/create";
+            return "user/order/payment";
```

Comment:
```suggestion
            return "user/order/new";
```
ri hợp lý hơn, đã order rồi còn payment nữa

mình theo chuẩn restful name đi gái